SwiftWisdom/Core/Geometry/Geometry.swift

```diff
@@ -86,6 +86,11 @@ public func random(range: Range<Int>) -> Int {
 }
 
 public func avg(numbers: CGFloat...) -> CGFloat {
+    return avg(numbers)
+}
+
+public func avg(numbers: [CGFloat]) -> CGFloat {
```

Comment:
The count of an empty array, which in essence is "no values" is always returned as a non-optional 0.  I think we should follow suit